packages/ember-glimmer/lib/helpers/-class.js

```diff
@@ -0,0 +1,25 @@
+import { InternalHelperReference } from '../utils/references';
+import { dasherize } from 'ember-runtime/system/string';
+
+function classHelper({ positional }) {
+  let path = positional.at(0);
+  let className = positional.at(1);
+  let value = path.value();
+
+  if (value === true) {
+    return dasherize(className.value());
```

Comment:
Wouldn't it be better to use className as it is when explicitly passed? (Ideally to use dasherized attribute name by default for helpers as well, but feels like it is not possible?)